public/data/harshraj8843.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Harsh Raj",
+    "bio": "Open Source ❤️ || Interested in Web Development",
+    "avatar": "https://avatars.githubusercontent.com/u/54644599?v=4",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/harshraj8843",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/harshraj8843",
+        "icon": "twitter"
+      }
+    ]
+  }
+  
```

Comment:
I would put this in another PR so it can be merged and not blocked by other changes that might need more discussion

Reply:
Thanks

Reply:
I agree with Eddie. 👍